jetstream/config.py

```diff
@@ -215,6 +225,19 @@ def status(self) -> Optional[str]:
         """
         return "Complete" if self.experiment_spec.end_date else self.experimenter_experiment.status
 
+    # Helpers for configuration templates
+    @property
+    def start_date_str(self) -> str:
+        if not self.start_date:
+            return "1970-01-01"
+        return self.start_date.strftime("%Y-%m-%d")
```

Comment:
I'm wondering, how likely is it that there is no start date? Also what do the `from_expr` usually look like (or more specifically what tables do they query)? My concern is that queries might end up $$$ if we query over a long time period.

Reply:
I think these are mostly draft experiments from the v1 API -- throwing an exception is probably also reasonable. We'd reject these in run() for lacking a start_date so it's probably fine to quit while resolving a configuration.

Writing this made me wish that we had an Experiment type without any Optional fields.

Reply:
Luckily, we have a new Exception type for just such a scenario...